Chapters/4.Syntax.md

```diff
@@ -39,8 +39,10 @@ the following grammar:
 <visit_ctxt> ::= "visit" "=" <identifier_core> ;
 <anchor_ctxt> ::= "anchor" "=" <identifier_core> ;
 <path_ctxt> ::= "path" "=" <path_absolute_escaped> ;
-<fragment_qualifier> ::= "lines" "=" <line_number> ["-" <line_number>] ;
-<line_number> ::= <dec_digit> + ;
+<fragment_qualifier> ::= 
+    "lines" "=" <dec_digits> ["-" <dec_digits>] 
+  |	"bytes" "="  <dec_digits> ["-" <dec_digits>] ;
+<dec_digits> ::= <dec_digit> + ;
```

Comment:
Minor suggestion: we should refactor this into a range definition, now that is used twice (for lines and for bytes).
Something like this (to be verified):
```
<fragment_qualifier> ::= "lines" "=" <dec_range> | "bytes" "=" <dec_range> ;
<dec_range> = <dec_digits> ["-" <dec_digits>] ;
```

Reply:
Thanks, done in 3a2b5c0a29696a4a96837116f0510eb177734a94